lib/board.rb

```diff
@@ -0,0 +1,70 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    reset!
+    #binding.pry
+  end
+
+  def reset!
+    @cells = Array.new(9, " ") #[" ", " ", " ", " ", " ", " ", " ", " ", " "]
+    #binding.pry
+  end
+
+  def display
+    counter = 0
+    @cells.each do |space|
+      counter += 1
+      if counter == 3 || counter == 6 || counter == 9
+        print " #{space} "
+      else
+        print " #{space} |"
+      end
+      if counter % 3 == 0
+        if counter == 9
+          break
+        else
+          print "\n"
+          puts "-----------"
+        end
+      end
+    end
+  end
+
+  def position(input)
+    input = input.to_i - 1
+    #binding.pry
+    return "#{@cells[input]}"
+  end
+
+  def full?
+    @cells.detect {|space| space == " "} ? false : true
```

Comment:
Use the most expressive and efficient iterator for a problem. `Board#full?` is asking if all elements are not empty strings. You can use logic that is symmetrical to those semantics with the [`#all?`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-all-3F) iterator.

`cells.all? {|cell| cell != " "}`

This is also more efficient in that the loop will stop after the iteration for which a cell is not empty.